### ISSUE TITLE
aria-required fails on combobox without aria-expanded

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-core#3090]
This should not fail as pointed out  in https://github.com/dequelabs/axe-core/issues/2505#issuecomment-881947373

```html
<div role="combobox"></div>
```

**Comment by padmavemulapati** [comment]
Validated with the latest code ,
its giving failure for the below cases:
`<div role="combobox"></div>`
and
`<div id="target" role="combobox" aria-expanded="true">expanded</div>
![image](https://user-images.githubusercontent.com/47409579/128319216-69fdd317-ccbb-4c40-87cb-24c191b57337.png)

and as per the issue, if we are not passing any `attributes` for `role=combobox` means it is in collapsed state only ,
so that the snippet `<div role="combobox"></div>` meaning is `expanded=false` , so that it should not fail
but its showing as failure
@straker , can you please confirm, is the fix applicable only when `aria-attributes` state is mentioned?

**Comment by straker** [comment]
@padmavemulapati no attributes does not equate to `expanded=false` and aria-expanded is a required attribute and `aria-expanded` does not have an implicit value defined (so no default) https://www.w3.org/TR/wai-aria-1.2/#combobox

**Comment by padmavemulapati** [comment]
@straker , Thank you for the clarification.

**Comment by straker** [comment]
No docs

**Comment by mshubat** [comment]
https://github.com/angular/components/issues/23570

This has been mentioned by the Angular team as being a false positive, as the aria-controls attribute is added once the dropdown for the select with role "combobox" is added to the dom. 

It seems as though this is still generating false positives. Any plans to address this?

Thanks.

**Comment by straker** [comment]
@mshubat The next axe-core release slated for the end of this month should include this fix.
